cli/tests/text_decoder_perf.js

```diff
@@ -0,0 +1,28 @@
+function r32() {
+  return Math.floor(Math.random() * 0xffffffff);
+}
+
+function generateRandomUTF8(size) {
```

Comment:
This function is wrong in countless ways.
Care to look again after a good night of sleep?